File: Armstrong.java
import java.util.Scanner;
public class Armstrong{
    public static void main(String[] args){
    Scanner input = new Scanner(System.in);
    System.out.print("Enter number of choice: ");
    int number = input.nextInt();

    int firstNumber = number / 100;
    int secondNumber = (number % 100) / 10;
    int thirdNumber = (number % 100) % 10;

    int multiple = 3;
    double cubeFirstNumber = Math.pow(firstNumber,multiple);
    double cubeSecondNumber = Math.pow(secondNumber,multiple);
    double cubeThirdNumber = Math.pow(thirdNumber,multiple);
    double sum = (cubeFirstNumber + cubeSecondNumber + cubeThirdNumber);
    System.out.println(sum);

    boolean equal = true;
    boolean notequal = false;
    if(number == sum){
    System.out.print(equal);
    }else{
    System.out.print(notequal);    
    }
}
}

    

File: armstrong.py
number = int(input("Enter number of choice: "))
first_number = int(number / 100)
second_number = int(number % 100) / 10
third_number = int(number % 100) % 10

cube_first_number = int(first_number ** 3)
cube_second_number = int(second_number ** 3)
cube_third_number = int(third_number ** 3)

sum = (int(cube_first_number + cube_second_number + cube_third_number))
print(sum)
same = True;
notsame = False;
if(number == sum):
    print(same)
else:
    print(notsame)
